src/iframeWrapper.ts

```diff
@@ -29,7 +29,8 @@ export default class IframeWrapper {
   constructor(
     private params: iframeWrapperParams,
     private iframeUrl: string,
-    private themeConfig: IWidgetThemeConfig
+    private themeConfig: IWidgetThemeConfig,
+    private destroyWalletUI: Function
```

Comment:
Using `Function` doesn't help with type-checking the function parameters or return value. Can we annotate them explicitly?

```suggestion
    private destroyWalletUI: (this: WalletProvider) => void
```

Reply:
working on it